### PR TITLE
188 aggregate results should prioritise convergence status over criteria

### PR DESCRIPTION
**Pull request by barondim**
**Checklist before requesting a review**

*use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes*

- [x] unit tests and non regression tests were added (update of algorithms)
- [ ] main documentation was updated (update of input/output file, update of algorithms)
- [ ] if this PR modifies a dictionary: the corresponding french dictionary was updated
- [ ] if this commit targets a release branch: the corresponding milestone was added in the ticket and in this PR

### CODE REVIEW

**Comment by barondim** [comment]
To be merged after https://github.com/dynawo/dynawo/pull/2688

**Comment by barondim** [comment]
- [ ] create nrt for LOCAL_VALUE and SUM criteria type

File: nrt/data/IEEE14/SA_convergence_failing_criteria_sum/result/aggregatedResults.xml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="ISO-8859-1" standalone="no"?>
```
Comment (rosiereflo) [inline nrt/data/IEEE14/SA_convergence_failing_criteria_sum/result/aggregatedResults.xml:1]: Is this file useful on github?

**Comment by barondim** [comment]
- [ ] Please, don't merge. NRTs fail.

**Review by rosiereflo — CHANGES_REQUESTED** [review]

**Review by rosiereflo — APPROVED** [review]

**Pull request merged**